db/re/item_db.txt

```diff
@@ -9897,7 +9897,7 @@
 18873,Sweet_Valentine_Out,Sweet Valentine Out,4,20,,100,,1,,0,0xFFFFFFFF,63,2,256,,1,1,865,{ bonus bMaxHPrate,7; bonus bMaxSPrate,7; },{},{}
 18874,One_Eyed_Glass_,Cyclops Glasses,4,10,,100,,,,1,0xFFFFFFFF,63,2,512,,1,,23,{},{},{}
 18877,Baron_Evil_Eye,Baron's Evil Eye,4,10,,300,,,,0,0xFFFFFFFF,63,2,512,,30,,989,{ bonus bDelayrate,-5; bonus bUseSPrate,5; },{},{}
-18878,Palace_Guard_Cap,Palace Guard Cap,4,10,,1000,,15,,1,0xFFFFFFFF,63,2,256,,50,1,991,{ .@r = getrefine(); bonus2 bAddEle,Ele_Neutral,4+.@r/6; bonus bLongAtkRate,4+.@r; bonus bUnbreakableHelm; },{},{}
+18878,Palace_Guard_Cap,Palace Guard Cap,4,10,,1000,,15,,1,0xFFFFFFFF,63,2,256,,50,1,991,{ .@r = getrefine(); bonus2 bSubEle,Ele_Neutral,5; bonus bLongAtkDef,5; bonus bUnbreakableHelm; if (.@r>5) { bonus2 bSubEle,Ele_Neutral,.@r-5; bonus bLongAtkDef,.@r-5; } },{},{}
```

Comment:
`.@val = max(5,getrefine()); bonus2 bSubEle,Ele_Neutral,.@val; bonus bLongAtkDef,.@val; bonus bUnbreakableHelm;` to simplify